### PR TITLE
Dynamodb Support for pg spanner

### PR DESCRIPTION
**Pull request by Deep1998**
Added changes to support dynamodb to pg spanner migration.

Manually tested for now.

### CODE REVIEW

File: sources/dynamodb/data.go [hunk 1]
```diff
@@ -149,22 +197,6 @@ func cvtColValue(attrVal *dynamodb.AttributeValue, srcType string, spType string
 				return nil, fmt.Errorf("failed to convert %v to a json string", attrVal.GoString())
 			}
 			return string(b), nil
-		case typeString:
-			return *attrVal.S, nil
-		case typeStringSet:
```
Comment (bharadwaj-aditya) [inline sources/dynamodb/data.go:154]: Do we retain the old functionality for non-pg spanner post these changes ?

Comment (Deep1998) [reply sources/dynamodb/data.go:154]: Yes we do. I made the changes accordingly to support both and integration tests for dydb passing means older version works fine.

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Comment by bharadwaj-aditya** [comment]
You will have some conflicts based on the other DynamoDb PR merged by Charvi. If there are significant changes, I will take another pass at this

**Pull request merged**